build_rpm.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+git checkout -- setup.cfg
+rm -rf build dist *.egg *.egg-info
+
+# Build community source tarball
+
+python setup.py sdist --formats=gztar
+
+# Build F5 rpm spec
+
+/bin/cp -f f5-setup.cfg setup.cfg
+
+python setup.py bdist_rpm --spec-only \
+  --release=1 \
+  --provides=f5-neutron-lbaas-dashboard \
+  --packager="Qin Zhao <q.zhao@f5.com>" \
+  --post-install=rpm_post_install.sh
+
+# Repackage source tarball
+
+cd dist
+
+VERSION=$(rpmspec -q --qf "%{version}" f5-neutron-lbaas-dashboard.spec)
+
+OLD=neutron-lbaas-dashboard-${VERSION}
+NEW=f5-neutron-lbaas-dashboard-${VERSION}
```

Comment:
how about using f5-openstack-dashboard, as we talked, the installed package name.